docs/README.md

```diff
@@ -42,6 +45,50 @@ unchanged. When a single number is given, the *maximum* dimension of the photo
 will be reduced to the given number, and the other dimension is chosen 
 according to the aspect ratio.
 
+EXIF/IPTC Tags
+--------------
+
+You can use the metadata embedded in the photos as a starter to fill out captions
+and copyright information. HugoPhotoSwipe will use this information when you run 
+`hps update` in the following manner:
+1. Prefer the information already specified in the album file
+2. Use the information in the metadata
+
+Using metadata requires a mapping for each field you want to be populated. Currently
+only caption and copyright are supported. In the configuration file, add the `tag_map`
+setting as follows:
+
+```yaml
+tag_map:
+  caption: exif.ImageDescription
+  copyright: exif.Artist
+```
+
+Caption will be saved to the album yaml file with the photo information. You can then
+edit it there if you wish. Because the album file is given priority, your changes will
+not be overridden, even after an update. Copyright is not yet used as this is currently 
+an album-level property.
+
+The format of the option is: `property: iptc/exif.tag`. Tag may include spaces. A full
```

Comment:
Suggest to spell it out a bit more: "`<property_name>: iptc.<tag_name>` or `<property_name>:exif.<tag_name>` (fo example: `copyright: exif.ImageDescription`).". The same goes for the error message if the tag is improperly formatted (in `_get_tag_value`)